fatek/symbol.py

```diff
@@ -74,7 +74,8 @@ def __calculate_offset(self):
         offset = self.offset_dict[register]
         if register == 'C' and current_value and number >= 200:
             number %= 200
-            offset = 9700 + 2 * (number)
+            number = number + number
+            offset = 9700
```

Comment:
Yeah, but first you wrote:

> Should the offset C255 with current_value=True be 9810 rather than 9865 ?

From where did you get the `9865` ? It looks good for me in the code, that was my point